iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -92,6 +92,9 @@ Fact.prototype.readableValue = function() {
             v = formattedNumber + " " + this.unit().valueLabel();
         }
     }
+    else if (this.isNil()) {
+        v = "nil";
```

Comment:
Move this above `isNumeric` so the `isNil` handling can be common?

Reply:
It's not quite that simple, as for numerics we still include units in the resulting string, and "nil" just replaces the numeric part, so we'd still need a second if.

Whether it makes sense to include units alongside "nil" is another question, but it's information that's in the report so I figure we should display it.

Reply:
Ah! I overlooked that.  Looks fine to me, thanks :-).